README.md

```diff
@@ -34,4 +34,5 @@ Please follow below steps to add new projects to the repository.
  - [Custom User Operation Event Listener](custom-user-operation-event-listener/) by [Nilasini Thirunavukkarasu](https://github.com/nilasini) - Ability to generate random passwords that satisfies your own password policies.
  - [Custom Scope Handler](custom-scope-handler/) by [Abilashini Thiyagarajah](https://github.com/Abilashini) - A custom scope handler to remove the scopes which are not allowed from the token request and generate the access token with only the allowed scopes. 
  - [Custom Basic Authenticator for Force Password Reset via OTP](custom-basic-authenticator-for-force-password-otp/) by [Dinali Dabarera](https://github.com/gdrdabarera) - A custom basic authenticator which does force password reset via OTP.
+ - [Custom Remote User Store Management](custom-remote-user-store-management/) by [Tharindu Bandara](https://github.com/tharindu-bandara) - An aggregator module to consume a remote user database as a secondary user store.
```

Comment:
How about __Custom Remote User Store Manager__? And the directory should be __custom-remote-user-store-manager/__

Reply:
Fixed in 51041691a23b06016ed6fa558504cc92b89c40b3 and 8a4b4fe793b231e0bd48d328c076d4cfd9494c5e